nn.py

```diff
@@ -207,23 +231,23 @@ def forward(self, x, num_augments=None):
 
         Args:
             x            = [torch.Tensor] images with shape (N, 1, SIZE, SIZE)
-            num_augments = [torch.Tensor] number of augmentations per sub-
-                                          problem with shape (BATCH_SIZE, 3)
+            num_augments = [torch.Tensor] number of augmentations per
+                sub-problem with shape (BATCH_SIZE, 3)
 
         Returns [torch.Tensor]*3:
             Non-normalized predictions for each class for each sub-problem.
         """
         # put images on GPU
         x = x.to(self.device)
 
-        # get correct number of channels for ResNet50
+        # get correct number of channels for ResNet-18
         h = self.conv1(x)
 
-        # get latent vectors from ResNet50
+        # get image embeddings from ResNet-18
         h = self.resnet18(h)
         h = h.flatten(start_dim=1)
 
-        # determine sub-problem logits
+        # determine predictions for each sub-problem
         h_graph, h_vowel, h_conso = _split_vectors(h, num_augments)
         y_graph = self.fc2(h_graph)
         y_vowel = self.fc3(h_vowel)
```

Comment:
Same as above: call these `fc1`, `fc2`, and `fc3`. (couldn't select the line with `self.fc4` in review mode)

Reply:
The problem with these is that the trained model, i.e. *model.pt* is expecting `fc2`, `fc3`, and `fc4` as I changed the documentation only recently. You can manually map the `fc2`, `fc3`, and `fc4` keys in the dictionary when calling `torch.load('model.pt')` to `fc1`, `fc2`, and `fc3` and then saving the dictionary again to *model.pt* to solve this issue.

Reply:
Ah okay, I'm not sure if that's worth the effort. It's not like it is a major issue so we could just leave it likes this as it is still quite clear what the layers are/mean.